src/Spain/Mews.Fiscalizations.Spain/Communication/SoapClient.cs

```diff
@@ -32,7 +32,8 @@ internal SoapClient(Uri endpointUri, X509Certificate certificate, TimeSpan httpT
             where TIn : class, new()
             where TOut : class, new()
         {
-            var messageBodyXmlElement = XmlSerializer.Serialize(messageBodyObject, new XmlSerializationData(GetSiiNameSpaces()));
+            var data = new XmlSerializationData(namespaces: GetSiiNameSpaces());
```

Comment:
Let's not be calling this data. If anything then xmlMetaData or something? 
`data`  kinda looks like the content. not just here, There's multiple places that do `var data` But we can discuss this with Marek as well.